pkg/storaged/block/format-dialog.jsx

```diff
@@ -271,8 +271,20 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
     else
         at_boot = "local";
 
-    const action_title = create_partition ? _("Create and mount") : _("Format and mount");
-    const action_variant = { tag: "nomount", Title: create_partition ? _("Create only") : _("Format only") };
+    let action_variants = [
+        { tag: null, Title: create_partition ? _("Create and mount") : _("Format and mount") },
+        { tag: "nomount", Title: create_partition ? _("Create only") : _("Format only") }
+    ];
+
+    const action_variants_for_empty = [
+        { tag: null, Title: create_partition ? _("Create") : _("Format") }
+    ];
+
+    if (client.in_anaconda_mode()) {
+        action_variants = [
+            { tag: "nomount", Title: create_partition ? _("Create") : _("Format") }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19807-20240103-141207-4bbc74c3-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)